doc/use_cli_effectively.md

```diff
@@ -125,22 +125,31 @@ If you are using az on a build machine, and multiple jobs can be run in parallel
        )
        az vm stop --ids %vm_ids% :: CLI stops all VMs in parallel 
 
-   ### Windows PowerShell scrips for saving to variables and using it later 
+### Windows PowerShell scrips for saving to variables and using it later
 
        $vm_ids=(az vm list -d -g my_rg --query "[?powerState=='VM running'].id" -o tsv)
        az vm stop --ids $vm_ids # CLI stops all VMs in parallel 
 
-   ### Windows batch scripts to loop through a list
+### Windows batch scripts to loop through a list
        ECHO OFF
        SETLOCAL
        FOR /F "tokens=* USEBACKQ" %%F IN (`az vm list -d -g my_rg --query "[?powerState=='VM running'].id" -o tsv`) DO (
            ECHO Stopping %%F
            az vm stop --ids %%F
        )
 
-   ### Windows PowerShell scrips to loop through a list
+### Windows PowerShell scrips to loop through a list
        $vm_ids=(az vm list -d -g my_rg --query "[?powerState=='VM running'].id" -o tsv)
        foreach ($vm_id in $vm_ids) {
            Write-Output "Stopping $vm_id"
            az vm stop --ids $vm_id
        }
+
+### CLI Environment Variables
+
+|  Environment Variable          | Description            |
+|--------------------------------|------------------------|
+| **AZURE_CONFIG_DIR**           | Global config directory for config files, logs and telemetry. If unspecified, defaults to `~/.azure`. |
+| **AZURE_EXTENSION_DIR**        | Extensions' installation directory. If unspecified, defaults to cliextensions directory within the global config directory. |
+| **AZURE_TEST_RUN_LIVE**        | If set to a truthy value (e.g. "True") runs tests live. |
```

Comment:
I suggest remove this and move them to the test section of command authoring. This doc is end users facing.

Reply:
okay will do, so otherwise am I missing any environment vars?